ferrocene/tools/generate-coverage-report/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
Are there plans to add more functionality in the `generate-coverage-report` tool? Otherwise we could inline its functionality in bootstrap itself.

Reply:
We can plan to add `llvm-cov` to generate report in the future using this tool. In that case, we have to handle the parameters to be passed between `grcov` and `llvm-cov`, this can be done using this tool.

So, we think it is better to have it as a tool.

Reply:
Hmm, I think it'd be better to pick between grcov and llvm-cov and stick to just one of them. This will be used to generate the reports to include in our documentation, and we're going to include a single copy anyway.

For the time being I'd still prefer if we remove the dedicate tool and invoke grcov/llvm-cov from bootstrap directly. Right now it just adds a level of indirection, and if in the future we'll reach a point where we need more complex logic we can revisit whether to have a separate tool.

Reply:
@pietroalbini We need your input for choosing between llvm-cov and grcov.

llvm-cov , produces more accurate data, but it does not have a way to skip lines/branch/region from the coverage report.

grcov, produces less accurate report (sometimes accounting the lines as not covered), but it has a way to skip lines from the coverage.

Reply:
I'd prefer llvm-cov then. Let's merge this PR and switch to llvm-cov in a future one.